server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetFieldMappingAction.java

```diff
@@ -33,6 +35,7 @@
 import static org.elasticsearch.rest.RestStatus.NOT_FOUND;
 import static org.elasticsearch.rest.RestStatus.OK;
 
+@ServerlessScope(Scope.PUBLIC)
```

Comment:
I think that this API could be left out.

Reply:
This is one that @tvernum had identified to be in, but that was just a first pass. I'll remove it so that the Search team can figure out what to do with it later.